docs/docs/releases/0.2.0.md

```diff
@@ -244,4 +244,114 @@ We also set up a workflow to reset the live demo regularly to mitigate reliabili
+
+#### If there is another database on the server
+
+If there is another PostgreSQL database on the same server, first log into that other database, and then run:
+
+```sql
+DROP DATABASE mathesar_django;
+CREATE DATABASE mathesar_django OWNER mathesar;  -- or whichever role you use to run Mathesar
+```
+
+#### If there is not another database on the server
+
+If there is **not** another database on the server, it's more difficult, since you can't drop the only database on a PostgreSQL server. First, log into the `mathesar_django` database, and run:
+```sql
+CREATE DATABASE mathesar_temp;
+```
+Log out, and log into the `mathesar_temp` database. Then run:
+```sql
+DROP DATABASE mathesar_django;
+CREATE DATABASE mathesar_django OWNER mathesar;  -- or whichever role you use to run Mathesar
+```
+Log out, and log into the `mathesar_django` database. Then run:
+```sql
+DROP DATABASE mathesar_temp;
+```
+
+After this, you're ready to proceed with the upgrade.
+
+### For installations using Docker Compose
+
+If you have a Docker compose installation (including one from the guided script), run the command below:
+
+```
+docker compose -f /etc/mathesar/docker-compose.yml up --pull always -d
+```
+
+### For installations done from scratch
+
+1. Go to your Mathesar installation directory
+
+    ```
+    cd xMATHESAR_INSTALLATION_DIRx
+    ```
+
+    !!! note
+        Your installation directory may be different from above if you used a different directory when installing Mathesar.
+
+1. Pull version 0.2.0 from the repository
+
+    ```
+    git pull https://github.com/mathesar-foundation/mathesar.git
+    git checkout 0.2.0
+    ```
+
+1. Update Python dependencies
+
+    ```
+    pip install -r requirements-prod.txt
+    ```
+
+1. Activate our virtual environment
+
+    ```
+    source ./mathesar-venv/bin/activate
+    ```
+
+1. You can skip the following if you're upgrading from versions 0.1.4 and above.
+    - If you're upgrading from versions <= 0.1.3, update your environment variables according to the [the new configuration specification](../administration/configuration.md#db).
+    - In particular, you must put the connection info for the internal DB into new `POSTGRES_*` variables. The `DJANGO_DATABASE_URL` variable is no longer supported.
+
+1. Add the environment variables to the shell before running Django commands
+
+    ```
+    export $(sudo cat .env)
+    ```
+
+1. Run Django migrations
+
+    ```
+    python manage.py migrate
+    ```
+
+1. Download and extract frontend assets
+
+    ```
+    wget https://github.com/mathesar-foundation/mathesar/releases/download/0.2.0/static_files.zip
+    unzip static_files.zip && mv static_files mathesar/static/mathesar && rm static_files.zip
+    ```
+
+1. Compile Mathesar translation files
+
+    ```
+    python manage.py compilemessages
+    ```
+
+1. Update Mathesar functions on the database:
```

Comment:
We don't install/update any functions using install.py anymore, I am skeptical whether we even need this step.

Reply:
You're right. Removed!